spec/caskinstaller_spec.rb

```diff
@@ -23,7 +25,9 @@ def do_install
         allow(Bundle::CaskInstaller).to receive(:installed_casks).and_return([])
         expect(Bundle).to receive(:system).with("brew", "install", "caskroom/cask/brew-cask").and_return(true)
         expect(Bundle).to receive(:system).with("brew", "cask", "install", "google-chrome").and_return(true)
+        expect(Bundle).to receive(:system).with("brew", "cask", "install", "firefox", "--appdir=/Applications").and_return(true)
         expect(do_install).to eq(true)
+        expect(do_install_with_args).to eq(true)
```

Comment:
Please put this call into a new test rather than adding to the existing one, thanks.